### DISCUSSION TITLE
(Feature request) Could you please advise on how to add multiple servers to one Trilium client, or how to set up two separate Trilium clients on Windows to connect to different servers?

### DISCUSSION THREAD

**Discussion by BLOSEregedit** [TriliumNext/Trilium#7360] [Q&A] [answered]
### Describe feature

Excuse me for disturbing you. I recently discovered your fantastic note-taking software and after trying it out, I'm delighted to say that it's the perfect note-taking tool that I've been searching for. While using and browsing the wiki, I learned that Trilium is primarily a personal knowledge management tool and does not support collaboration or real-time operations. As there are two of us, I thought about creating two servers on an intranet NAS using Docker, so that each of us can have our own server to use. I have implemented this method, but I have a scenario where I want to connect to both databases simultaneously from a Windows computer on the intranet NAS. I searched and found that old versions of Trilium support multi-source synchronization, but in the new version, I didn't find any way to configure this. Then I thought, could I unzip two Windows clients and connect them to two servers separately? However, I couldn't find any tutorial on how to configure this or the corresponding config file. So I'm here to ask if there's a way to meet my needs: one Windows computer, one or two clients connected to two servers respectively?

**Comment by zadam** [comment, marked as answer]
One client can connect to one server only.

You can have multiple separate trilium instances on one computer where each one will connect to a different server instance.

The key to do this is to configure environment variables `TRILIUM_PORT` and `TRILIUM_DATA_DIR` so that each instance runs on a different point and saves data to a different path.

Some info: https://github.com/zadam/trilium/wiki/Data-directory

In your case I would create 2 .bat files which would set the env variables for each instance. You can check [trilium-portable.bat](https://github.com/zadam/trilium/blob/master/bin/tpl/trilium-portable.bat) for inspiration.
